src/main/java/KgGm.java

```diff
@@ -0,0 +1,14 @@
+public class KgGm {
+    public static void kilogram(int kg) {
```

Comment:
use camelCase names for the methods and it should be a verb.
e.g. convertKilogramToGrams